app/components/layout/FindHeader.scss

```diff
@@ -15,25 +15,67 @@
     padding: calc-em(20px) 0 calc-em(65px);
   }
 
-  h1 {
-    @extend %font-size-hero;
-    @extend %base-max-width;
-    font-weight: 600;
-    color: $color-grey8;
-    text-align: center;
-    margin-bottom: calc-em(12px);
+  .header-container {
+    @extend %search-max-width;
+    display: flex;
     @include r_max($break-tablet-s) {
       width: 100%;
-      margin-bottom: calc-em(18px);
-      max-width: 20rem;
+      padding: 0 calc-em(20px) 0 calc-em(20px);
+    }
+
+    .header-text {
+      width: 70%;
+      @include r_max($break-tablet-s) {
+        width: 100%;
+      }
+
+      p {
+        color: $color-grey5;
+      }
+
+      h1 {
+        @extend %font-size-hero;
+        font-weight: 600;
+        color: $color-grey8;
+        @include r_max($break-tablet-s) {
+          width: 100%;
```

Comment:
Is this necessary to set? Block elements by default span the width of their parent.